src/codegen/codegen_call.rs

```diff
@@ -94,37 +94,11 @@ pub fn codegen_call<'a>(
     }
 
     match call_operation_name_str {
-        "llvm.sadd.with.overflow.i32" => {
-            let operand1_name = get_var_name(
-                &instruction.get_operand(0).unwrap().left().unwrap(),
-                &solver,
-                namespace
-            );
-            let operand2_name = get_var_name(
-                &instruction.get_operand(1).unwrap().left().unwrap(),
-                &solver,
-                namespace
-            );
+        s if s.starts_with("llvm.sadd.with.overflow.i") => {
+            // Translate the intrinsic integer size to an i64 representing the min/max representable numbers
+            let s_size = &s[25..];
```

Comment:
I'm not the biggest fan of the 25 constant but I guess it's not too bad because it's right next to the code (if condition) that checks the size 25 prefix `llvm.sadd.with.overflow.i` of the string.